[EDU-4929] feature: add Docusaurus Material UI guide (#1120)
aziontech/docs: src/content/docs/en/pages/main-menu/additional-resources/templates-and-integrations/understand-azion-templates.mdx

<LinkButton link="/en/documentation/products/guides/azion-starter-kit/" label="go to the Azion Starter Kit guide" severity="secondary" /> 

> #### Docusaurus Blog

Author: ```suggestion
#### Docusaurus Blog Starter Kit
```

Create and manage a blog page with support to Markdown, MDX, and React components, based on Docusaurus.

> <LinkButton link="/en/documentation/products/guides/docusaurus-blog-template/" label="go to the Docusaurus Blog guide" severity="secondary" /> 

Author: ```suggestion
<LinkButton link="/en/documentation/products/guides/docusaurus-blog-starter-kit/" label="go to the Docusaurus Blog Starter Kit guide" severity="secondary" /> 
```

> #### Docusaurus Blog Starter Kit
> 
> Create and manage a blog page with support to Markdown, MDX, and React components, based on Docusaurus.
> 
> <LinkButton link="/en/documentation/products/guides/docusaurus-blog-starter-kit/" label="go to the Docusaurus Blog Starter Kit guide" severity="secondary" /> 
> 

Author: ```suggestion
```
